src/basics/building-deploying-interfacing.md

```diff
@@ -0,0 +1,133 @@
+# Building, deploying and interfacing
+
+Now that all of the dependencies are installed and we have successfully chosen a [development track], we can finally discuss program development, including compiling, deploying and interacting with it.
+
+The `arch-cli` comes with a demo dapp called [GraffitiWall]; each message written to the wall contains a timestamp, name and note.
+
+Find the program's logic within the `src/app/program/src/lib.rs` file.
+
+### Build
+Fortunately, the program was already compiled during the [initialize] step. If we wish to modify the program further before deploying, we would need to rebuild the program manually. If you do not wish to modify the program, skip this step and move to [Deploy].
```

Comment:
Is this true? Interesting, I didn't even realize that.

Reply:
It looks like not anymore. Just updated the output to reflect this and will change this as well.